src/elements/core/base-elements/mini-button-base-element.ts

```diff
@@ -0,0 +1,16 @@
+import type { TemplateResult } from 'lit';
+
+import { SbbIconNameMixin } from '../../icon.js';
+import { slotState } from '../decorators.js';
+import { SbbNegativeMixin } from '../mixins.js';
+
+import { SbbButtonBaseElement } from './button-base-element.js';
+
+@slotState()
+export abstract class SbbMiniButtonBaseElement extends SbbNegativeMixin(
+  SbbIconNameMixin(SbbButtonBaseElement),
```

Comment:
We must never import from outside the core directory from inside the core directory.
I would suggest moving this base element to the mini button.
What do you think?